vgen/src/main/java/io/gapi/vgen/csharp/CSharpGapicCodeLanguageProvider.java

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gapi.vgen.csharp;
+
+import com.google.api.tools.framework.model.Interface;
+import com.google.api.tools.framework.model.Model;
+
+import io.gapi.vgen.ApiConfig;
+
+public class CSharpGapicCodeLanguageProvider extends CSharpGapicLanguageProvider<Interface> {
+
+  private final CSharpInterfaceViewing viewing = new CSharpInterfaceViewing();
```

Comment:
To me it doesn't really make sense to make a subclass purely for returning a different concrete Viewing that the parent class uses. This approach has created a bunch of trivial concrete classes. I think that the correct viewing should just be created at instantiation time and passed into the existing <Language>GapicLanguageProvider class, which should hold the instance for itself.